homeassistant/components/vera/config_flow.py

```diff
@@ -1,25 +1,47 @@
 """Config flow for Vera."""
 import pyvera as pv
 from requests.exceptions import RequestException
+import voluptuous as vol
 
 from homeassistant import config_entries
+from homeassistant.helpers import config_validation as cv
 
-from .const import CONF_CONTROLLER, DOMAIN  # pylint: disable=unused-import
+from .const import CONF_CONTROLLER, DOMAIN
 
 
 class VeraFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     """Vera config flow."""
 
+    async def async_step_user(self, config: dict = None):
+        """Handle user initiated flow."""
+        if self.hass.config_entries.async_entries(DOMAIN):
```

Comment:
If we plan to allow more than one config entry, we should try to set a unique id for the config entry. That's a new feature.

https://github.com/home-assistant/home-assistant/blob/16199b7a52146c563de36ce5e50cd1b59dbed03c/homeassistant/config_entries.py#L819-L847

Reply:
I think we can remove this check if we set a unique id in the finish step.

Reply:
True, the unique check is performed in the finish step. However, this means the user as entered information only to be told you cannot do that. It's not a good user experience. This approach tells the user they cannot add another before entering information.